test/_testServer.js

```diff
@@ -0,0 +1,11 @@
+"use strict";
+
+var jsonApiTestServer = require("jsonapi-server/example/server.js");
+if (!jsonApiTestServer.start) return;
```

Comment:
When browserify blanks a require via the `--ignore` param, it can still be `require()d` but it rocks up as `{ }` - if that is the case, don't attempt to start+stop the test server, because we're browserside and theres a real server instance to target.